api/client/task/print.go

```diff
@@ -41,7 +41,7 @@ func (t tasksBySlot) Less(i, j int) bool {
 }
 
 // Print task information in a table format
-func Print(dockerCli *client.DockerCli, ctx context.Context, tasks []swarm.Task, resolver *idresolver.IDResolver) error {
+func Print(dockerCli *client.DockerCli, ctx context.Context, tasks []swarm.Task, resolver *idresolver.IDResolver, noTrunc bool) error {
```

Comment:
should we inverse this, and make this `trunc`? Maybe just preference, but not really fond of the double negative :D

happy to hear what others think